examples/SampleApp/src/screens/ChannelListScreen.tsx

```diff
@@ -76,6 +76,8 @@ export const ChannelListScreen: React.FC = () => {
   } = useTheme();
 
   const searchInputRef = useRef<TextInput | null>(null);
+  const scrollRef = useRef<FlatList<any>>(null);
```

Comment:
Same as above, `any` doesn't seem right to use here to me

Reply:
Even I would prefer not using `any`, what could be a better option to replace `any`?

Reply:
I think you can do `FlatList<Channel> | null`

Reply:
Similar to this - https://github.com/GetStream/stream-chat-react-native/blob/develop/package/src/components/ChannelList/ChannelList.tsx#L370